src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -670,7 +670,7 @@ function automorphism_group(K::SimplicialComplex; action=:on_vertices)
 end
 
 @doc raw"""
-    on_simplicial_complex(K::SimplicialComplex, g::PermGroupElem)
+    on_simplicial_complex(K::SimplicialComplex, g::PermGroupElem; action=:on_vertices)
```

Comment:
is this maybe a conflict resolution artifact? it does not match the signature of the function this docstring is attached to